Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -189,22 +195,69 @@ void WiiPane::CreateWiiRemoteSettings()
   wii_remote_settings_group_layout->addWidget(m_wiimote_motor, 3, 0, 1, -1);
 }
 
-void WiiPane::OnEmulationStateChanged(bool running)
+void WiiPane::CreateWhitelistedUSBPassthroughDevices()
```

Comment:
Why did you move this code around? It has no relevance to your PR.

Reply:
It's just confusing a bit when similar panes have different methods of achieving the same exact thing, such as method names and sorting. Yes this would fit better in a code cleanup PR.